packages/apps/job-launcher/server/src/common/enums/job.ts

```diff
@@ -4,6 +4,7 @@ export enum JobStatus {
   LAUNCHED = 'LAUNCHED',
   COMPLETED = 'COMPLETED',
```

Comment:
I think in database we are not going to have COMPLETED status, so we can remove this

Reply:
For the current implementation, I agree, but this status will be used when we start tracking the escrow with the status [EscrowStatuses.Paid](https://github.com/humanprotocol/human-protocol/blob/c7648cd7b48b0cc086c9de9bcc4ff3b877a99477/packages/core/contracts/interfaces/IEscrow.sol#L6), at least all completed jobs on the launcher side will be have [LAUNCHED](https://github.com/humanprotocol/human-protocol/blob/c7648cd7b48b0cc086c9de9bcc4ff3b877a99477/packages/apps/job-launcher/server/src/common/enums/job.ts#L1) status. Any ideas?

Reply:
I would not use status from database apart from the launching process. For the rest I would use subgraph data.